app.config.yaml

```diff
@@ -38,6 +38,13 @@ application:
               authToken: ${AEM_ADMIN_API_AUTH_TOKEN}
             annotations:
               final: true
+          get-all-skus:
```

Comment:
maybe we name this `fetch-all-products` ?

Reply:
I mean the name only, to make it more generic (as we technically get the skus + url keys, not the skus only)